README.md

```diff
@@ -29,18 +30,18 @@ npm install @pinecone-database/pinecone
 
 ### Initializing the client
 
-There are two pieces of configuration required to use the Pinecone client: an API key and environment value. These values can be passed using environment variables or in code through a configuration object. Find your configuration values in the console dashboard at [https://app.pinecone.io](https://app.pinecone.io).
+There is one piece of configuration required to use the Pinecone client: an API key. This value can be passed using an environment variable or in code through a configuration object. Find your API key in the console dashboard at [https://app.pinecone.io](https://app.pinecone.io).
 
 #### Using environment variables
 
 The environment variables used to configure the client are the following:
 
 ```bash
 PINECONE_API_KEY="your_api_key"
-PINECONE_ENVIRONMENT="your_environment"
+PINECONE_CONTROLLER_HOST="your_environment"
```

Comment:
Is the for the environment? Seems like something most users won't use so maybe we can bury?

Reply:
I had that question too, I think it's fine burying.